_build/data/docs/commandline.rst

```diff
@@ -26,37 +53,56 @@ The EventGhost main executable accepts the following command line arguments:
     Shorter alias for the :option:`-hide` option.
 
 
-.. cmdoption:: -file <filename>
+.. cmdoption:: -multiload
 
-    Opens the XML configuration file <filename> on start-up (instead of the 
-    last loaded file). 
+    Allow multiple sessions of EventGhost on the same computer.
```

Comment:
If the first session of EG has been run with elevated privileges all subsequent sessions bust have elevated permissions as well